### ISSUE TITLE
Integration with modern build processes?

### ISSUE THREAD

**Issue by antoligy** [formvalidation/support#440]
I'm really liking the look of FormValidation, however it doesn't look like it's possible to use this in a build process as there is no module (ES6/CommonJS/AMD) support, let alone Bower/NPM packages - Is this something that is on your roadmap?

**Comment by nghuuphuoc** [comment]
Yes, it's on roadmap. Supporting these kind of modules will be available in next major v0.8.0.

**Comment by Steven-Pingel** [comment]
Is there a planned / public eta on v0.8.0?

**Comment by jacobkossman** [comment]
has this been added in v0.8.0?
